core/src/main/java/com/linecorp/armeria/client/UserClient.java

```diff
@@ -163,22 +161,19 @@ protected final O execute(@Nullable EventLoop eventLoop,
         }
 
         try (SafeCloseable ignored = RequestContext.push(ctx)) {
```

Comment:
I wasn't able to use UserClient in gRPC since it wasn't designed for streaming.

Can you update this, or maybe we can move to a request context constructor?

https://github.com/line/armeria/blob/master/grpc/src/main/java/com/linecorp/armeria/client/grpc/ArmeriaChannel.java